### PR TITLE
Update debugging.md

### PR DESCRIPTION
**Pull request by Nigusu-Allehu**
# Bug

Fixes: 

## Description
Just makes sure we have https://github.com/NuGet/NuGet.Client/pull/5981 documented
## PR Checklist

- [x] Meaningful title, helpful description and a linked NuGet/Home issue
- [ ] Added tests
- [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.

### CODE REVIEW

**Review by nkolev92 — DISMISSED** [review]

File: docs/debugging.md [hunk 1]
```diff
@@ -125,13 +125,9 @@ If you want to test dotnet.exe explicitly, so you don't have to worry about whet
 
 ### Debugging NuGet Command Xplat Functionality (add-package/remove-package/list package)
 
-Functionality such as `dotnet.exe add package` or `dotnet list package`, is implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```
Comment (zivkan) [inline docs/debugging.md:128]: I think this sentence is still valuable to keep, especially if we take my suggested heading change.  I'd go even one further and change it to:

```suggestion
All of the `dotnet nuget` commands, in addition to some others, such as `dotnet add package`, dotnet list package`, and `dotnet search`, are implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```

File: docs/debugging.md [hunk 2]
```diff
@@ -125,13 +125,9 @@ If you want to test dotnet.exe explicitly, so you don't have to worry about whet
...
-
-To debug with `MSBuildLocator`, you need to un-comment the `PackageReference` for `Microsoft.Build.Locator` in `NuGet.CommandLine.XPlat.csproj`. Additionally, un-comment the code utilizing `MSBuildLocator` in that project's `Program.cs`.
-
 There are 2 ways to debug this project:
 
-* Given that [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj) is an exe, you can set it as the startup project and run it as you would any other command line project in Visual Studio. Note that some commands list arguments in a different order to the dotnet cli.
+* Given that [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj) is an exe, you can set it as the startup project and run it as you would any other command line project in Visual Studio. However, if that is not working, please make sure `UseMSBuildLocator` is set to true. **Note**: some commands list arguments in a different order to the dotnet cli.
```
Comment (zivkan) [inline docs/debugging.md]: Markdown only adds new lines into the rendered HTML when there's an empty line, so we can have 1 sentence per line in the markdown source. This makes diffs smaller in the future when lines like this get changed, and it makes it easier to comment on specific sentences within a paragraph.

```suggestion
* Given that [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj) is an exe, you can set it as the startup project and run it as you would any other command line project in Visual Studio.
   However, if that is not working, please make sure `UseMSBuildLocator` is set to true.
   **Note**: some commands list arguments in a different order to the dotnet cli.
```

File: docs/debugging.md [hunk 3]
```diff
@@ -125,13 +125,9 @@ If you want to test dotnet.exe explicitly, so you don't have to worry about whet
 
 ### Debugging NuGet Command Xplat Functionality (add-package/remove-package/list package)
```
Comment (zivkan) [inline docs/debugging.md]:
```suggestion
### Debugging NuGet's integrated `dotnet` CLI commands
```

**Review by zivkan — DISMISSED** [review]

File: docs/debugging.md [hunk 4]
```diff
@@ -123,15 +123,14 @@ The easiest way to test the pack functionality with dotnet.exe is to install the
 
 If you want to test dotnet.exe explicitly, so you don't have to worry about whether you installed the correct package in the project, refer to [Patching dotnet.exe to test the NuGet functionality](#patching-dotnetexe-to-test-the-nuget-functionality).
 
-### Debugging NuGet Command Xplat Functionality (add-package/remove-package/list package)
-
-Functionality such as `dotnet.exe add package` or `dotnet list package`, is implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
-
-To debug with `MSBuildLocator`, you need to un-comment the `PackageReference` for `Microsoft.Build.Locator` in `NuGet.CommandLine.XPlat.csproj`. Additionally, un-comment the code utilizing `MSBuildLocator` in that project's `Program.cs`.
+### Debugging NuGet's integrated `dotnet` CLI commands
+All of the `dotnet nuget` commands, in addition to some others, such as `dotnet add package`, dotnet list package`, and `dotnet search`, are implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```
Comment (loic-sharma) [inline docs/debugging.md]:
```suggestion
All of the `dotnet nuget` commands, in addition to some others, such as `dotnet add package`, `dotnet list package`, and `dotnet search`, are implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```

File: docs/debugging.md [hunk 5]
```diff
@@ -123,15 +123,14 @@ The easiest way to test the pack functionality with dotnet.exe is to install the
 
 If you want to test dotnet.exe explicitly, so you don't have to worry about whether you installed the correct package in the project, refer to [Patching dotnet.exe to test the NuGet functionality](#patching-dotnetexe-to-test-the-nuget-functionality).
 
-### Debugging NuGet Command Xplat Functionality (add-package/remove-package/list package)
-
-Functionality such as `dotnet.exe add package` or `dotnet list package`, is implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
-
-To debug with `MSBuildLocator`, you need to un-comment the `PackageReference` for `Microsoft.Build.Locator` in `NuGet.CommandLine.XPlat.csproj`. Additionally, un-comment the code utilizing `MSBuildLocator` in that project's `Program.cs`.
+### Debugging NuGet's integrated `dotnet` CLI commands
+All of the `dotnet nuget` commands, in addition to some others, such as `dotnet add package`, `dotnet list package`, and `dotnet search`, are implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```
Comment (zivkan) [inline docs/debugging.md:128]: I'm pretty sure that if you install a markdown linter, it'll suggest to you to have a line of whitespace both before and after headings.

**Review by zivkan — DISMISSED** [review]

**Review by zivkan — APPROVED** [review]

**Pull request merged**